### ISSUE TITLE
Cursor disappears after running `spec`

### ISSUE THREAD

**Issue by nilsreichardt** [invertase/spec#46]
## Demo

https://user-images.githubusercontent.com/24459435/198004003-201f9d57-539f-4f12-b39b-154622b59cad.mov

## Steps to reproduce

```sh
dart pub global activate spec_cli
flutter create temp
cd temp
spec
```

## Additional context

* MacBook Pro M1
* macOS 12.6

**Comment by rrousselGit** [comment]
Did you perform an early abort or something?

That's not the behavior I get.

**Comment by nilsreichardt** [comment]
No, I'm pressing nothing 🤔 Unfortunately, I'm not able to go later than `0.1.3` because of the Riverpod issue :|

**Comment by bartekpacia** [comment]
I've also started experiencing this problem recently. Executing `tput cvvis` restores the cursor, but... yeah, strange.

**Comment by nilsreichardt** [comment]
Using `spec --watch` and removing the following line works for me:

https://github.com/invertase/spec/blob/7f6ec1a2bf61edcad933689b6dc1dcfee7dce238/packages/spec_cli/lib/src/command_runner.dart#L177

**Comment by nilsreichardt** [comment]
Since a short time, I have the issue with `dart run build_runner watch` 🤔
